crates/worktree/src/worktree.rs

```diff
@@ -6157,6 +6151,11 @@ fn git_status_to_proto(status: GitFileStatus) -> i32 {
     }
 }
 
+async fn canonicalize(fs: Arc<dyn Fs>, path: &Path) -> Result<PathBuf> {
```

Comment:
Can we make it look like

```suggestion
async fn full_path(fs: &Fs, path: &Path) -> Result<PathBuf> {
```

?

Would love a name, better than `full_path` so any suggestions welcome.
`canonicalize` is sort of overloaded by the existence of the `fs.canonicalize` and I'd love to avoid it.